wherehows-frontend/conf/routes

```diff
@@ -140,17 +140,9 @@ POST          /api/v1/datasets/:id/watch
 
 DELETE        /api/v1/datasets/:id/watch/:watchId                               controllers.api.v1.Dataset.unwatchDataset(id:Int, watchId:Int)
 
-GET           /api/v1/datasets/:id/compliance                                   controllers.api.v1.Dataset.getDatasetCompliance(id:Int)
+GET           /api/v1/datasets/:id/privacy                                      controllers.api.v1.Dataset.getDatasetCompliance(id:Int)
```

Comment:
Yes, I was going to talk to @theseyi about this. Currently we are using this privacy API, it makes more sense to rename to compliance, but the UI needs to rename the API calls also.